runtime/compiler/build/toolcfg/gnu-xlc/common.mk

```diff
@@ -108,6 +108,16 @@ endif
 
 ifeq ($(BUILD_CONFIG),prod)
     CX_DEFINES+=$(CX_DEFINES_PROD)
+    ifeq (,$(findstring ppc64le,$(PLATFORM)))
+         # big endian
+        C_FLAGS+=-qdebug=nscrep
+        # work-around for XL C/C++ 13.1 compiler bug
```

Comment:
@gita-omr are you happy with this comment?

Reply:
The comment is fine, but now I have worries about  -qdebug=nscrep. It's a bit hard to see the net change in the github... Why was it needed?

Reply:
I found this reference. 126996 refers to compjazz.
`126996: Apply -qdebug=nscrep to r14/open/dev and remove -qdebug=nipra from open/dev - Rid of nipra and apply nscrep debug option to xlC	Julian Wang	Mar 11, 2016`
```
xlC bugs:   pmr 40589 leads to the necessity of -qdebug=nscrep for the time being;
At the same time, V13 disabled ipra anyway. We can remove -qdebug=nipra from releases using V13.
```